numpy/core/fromnumeric.py

```diff
@@ -952,6 +958,10 @@ def argmax(a, axis=None, out=None):
     >>> np.argmax(a, axis=1)
     array([2, 2])
 
+    Indices of the maximal elements of a N-dimensional array:
+    >>> np.unravel_index(np.argmax(a, axis=None), a.shape)
+    (1, 2)
```

Comment:
Would be nice to show that the result of this can be used to index directly into `a`, and the same for the argsort example